data-review/114-celver-scenario/all_ghg_emissions_by_source.metadata.json

```
@@ -0,0 +1,361 @@
+{
+ "id": "https://openenergyplatform.org/dataedit/view/model_draft/all_ghg_emissions_by_source",
+ "@id": "https://openenergyplatform.org/dataedit/view/model_draft/all_ghg_emissions_by_source",
+ "name": "model_draft/all_ghg_emissions_by_source",
```

`model_draft/` ?

---

Thanks for the review :) Correct, ‘model_draft/’ is not part of the name as it describes the data theme and can change frequently, it is only relevant in the id field and later also in the resources field where the database structure becomes relevant.

```suggestion
 "name": "all_ghg_emissions_by_source",
```